### DISCUSSION TITLE
pip3.9 not found even after installing it. Create jobs failed

### DISCUSSION THREAD

**Discussion by Pappik** [learndevopsonline/learndevopsonline#1658] [Q&A] [answered]
tried multiple ways in installing pip but still failed. Checked all such discussions, but still no improvement.
<img width="673" alt="Screenshot 2023-12-04 222746" src="https://github.com/learndevopsonline/learndevopsonline/assets/89238944/ede1876e-9f7e-46c1-9d12-15b03cd86ba1">
workstation: 54.161.51.128
jenkins: 54.84.165.65

**Comment by sreedharm07** [comment]
@Pappik 
install python-jenkins in it, it will work
search in labauto you will find that.

**Reply by Pappik** [reply]
tq fr ur reply @sreedharm07 . i dont find python-jenkins in labauto, can see just jenkins. could you please work on this. might be a wrong approach but spending hours on this.

**Reply by sreedharm07** [reply]
ok i think its not there 
just do yum install python-jenkins

**Reply by sreedharm07** [reply]
sudo pip3.11 or what ever your version do that (sudo pip3.11 install python-jenkins)
it seems yours was 3.9 so do pip3.9

**Reply by Pappik** [reply]
yeah did all of them. even tried "sudo pip3.9/3.11 install python-jenkins/python3-jenkins" too

**Reply by sreedharm07** [reply]
ok please give your ip here
jenkins and workstation and jenkins uname and pswd

**Reply by Pappik** [reply]
workstation: 54.161.51.128
jenkins: 54.84.165.65.  user: admin, passwd: admin123

**Reply by sreedharm07** [reply, marked as answer]
@Pappik 
your all jobs in jenkins should run in workstation only
but yours was running in builtin-node please configure agent properly,
then it will works 
if you want to run in buildin node only menas then install python-jenkins in jenkins server 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/1401c9c5-c81f-4541-8d20-a86885d1db5d)

**Reply by Pappik** [reply]
ok i wil check it @sreedharm07 and get back for any other issues. its really a great help. tq
